### PR TITLE
CSSWA-524

### PR DESCRIPTION
**Pull request by adtandale**

### CODE REVIEW

**Comment by adtandale** [comment]
Please review.

File: piqe_ocp_lib/api/resources/ocp_base.py [hunk 1]
```diff
@@ -95,15 +94,17 @@ def _get_ocp_version(self):
...
             client = self.dyn_client.resources.get(api_version="config.openshift.io/v1", kind="ClusterVersion")
-
             version = client.get(name=CLUSTER_VERSION_OPERATOR_ID)
         except ApiException as e:
             logger.exception(f"Exception was encountered while trying to obtain cluster version: {e}")
             return None
-
+         
         version_query = "sort_by(status.history[?state=='Completed'], &completionTime)[::-1].version"
+        print(version_query)
```
[the remark this answers is not in the document]
Comment (adtandale) [reply piqe_ocp_lib/api/resources/ocp_base.py]: Removed.

**Review by btjd — APPROVED** [review]

**Comment by btjd** [comment]
LGTM

**Comment by amacmurr** [comment]
LGTM

**Pull request merged**